### PR TITLE
Feat/support local hugging face models

### PR DESCRIPTION
**Pull request by saattrupdan**
This implements evaluation of local Hugging Face models (saved with the `save_pretrained` method, for instance). 

Following the other model sources, this is located in a separate module, `local_hf_utils`, with the following three functions:
- `load_local_hf_model`, which loads and sets up a local HF model.
- `hf_model_exists_locally`, which checks whether a model exists. It does this by checking several things:
    - That the model folder exists.
    - That the configuration can be loaded using `AutoConfig.from_pretrained`.
    - That the folder contains a model, being a file with the file format `.bin`, `.pt` or `.msgpack`.
- `get_hf_model_config_locally`, which builds a `ModelConfig` object with the relevant information. The label conversion mappings are retrieved from the model's HF config. The framework is retrieved by looking at file formats:
    - It looks for `.bin` and `.pt` files for PyTorch models.
    - It looks for `.msgpack` files for JAX models.
    - It looks for `.h5` files for TensorFlow models (we don't support TF, so this will throw an informative error)
    - If none of the above exist, an error is thrown.

### CODE REVIEW

**Review by AJDERS — APPROVED** [review]
Besides possible conflict, LGTM!
